ledger/complete/mtrie/trie/trie.go

```diff
@@ -94,6 +94,9 @@ func (mt *MTrie) RootNode() *node.Node {
 // String returns the trie's string representation.
 // Concurrency safe (as Tries are immutable structures by convention)
 func (mt *MTrie) String() string {
+	if mt.IsEmpty() {
+		return fmt.Sprintf("Empty Trie with default root hash: %x\n", mt.RootHash())
+	}
```

Comment:
I think some logic for the empty trie is necessary now. Otherwise, `String()` panics for the empty trie:
https://github.com/onflow/flow-go/blob/85c924562513988126f3d01e70dae4d0d9cfd37d/ledger/complete/mtrie/trie/trie.go#L101

Reply:
Ah my bad, I though you were worried about the function `mt.RootHash()` being the issue, I forgot about the other `Fmt.Str`